src/Sylius/Bundle/AdminBundle/Resources/assets/scripts/autocomplete-xss-protection.js

```
@@ -0,0 +1,43 @@
+document.addEventListener('autocomplete:pre-connect', (event) => {
+    const options = event.detail.options;
+
+    if (!options.render) {
+        return;
+    }
+
+    const labelField = options.labelField || 'text';
+
+    const wrapRenderer = (renderer) => {
+        if (!renderer) return renderer;
+        return (data, escape) => {
```

Biome (2.4.6)

[error] Do not shadow the global "escape" property.

(lint/suspicious/noShadowRestrictedNames)